### PR TITLE
Patches needed for the rest of tests

### LINKED ISSUE #17: Patches needed for remainder of testing
Some patches are needed to port the remaining tests to the new Jenkins.

### PR DESCRIPTION
**Pull request by brianjmurrell**
Fixes #17

These patches are needed so that CI testing works on the new jenkins.

### CODE REVIEW

File: chroma-manager/tests/framework/integration/installation_and_upgrade/jenkins_steps/main [hunk 1]
```diff
@@ -6,7 +6,7 @@
 set_defaults true
 check_for_autopass
 
-export CLUSTER_CONFIG_TEMPLATE=${CLUSTER_CONFIG_TEMPLATE:-"chroma/chroma-manager/tests/framework/integration/shared_storage_configuration/full_cluster/shared_storage_configuration_cluster_cfg.json"}
+export CLUSTER_CONFIG_TEMPLATE=${CLUSTER_CONFIG_TEMPLATE:-"$CHROMA_DIR/chroma-manager/tests/framework/integration/shared_storage_configuration/full_cluster/shared_storage_configuration_cluster_cfg.json"}
```
Comment (jgrund) [inline chroma-manager/tests/framework/integration/installation_and_upgrade/jenkins_steps/main]: Where is `$CHROMA_DIR` being set?

File: chroma-manager/tests/framework/integration/shared_storage_configuration/simulator/cluster_setup [hunk 2]
```diff
@@ -98,25 +98,25 @@ EOF
 
 # Copy the repo to the test node (don't want to have to clone every test run - impacts Gerrit)
 # We exclude chroma-externals and copy just what we need to reduce the io load on our builders.
-TEST_DEPS="chroma/chroma-manager/tests/framework/integration/shared_storage_configuration/simulator/test_dependencies"
-(tar -czf - --exclude='chroma/chroma-externals*' .; tar -czf - $(sed -e 's|^|chroma/chroma-externals/|' < $TEST_DEPS); tar -czf - chroma/chroma-externals/*.zip chroma/chroma-externals/*.tar.*) | ssh chromatest@$CHROMA_MANAGER "mkdir -p chroma_test_env; tar -C chroma_test_env -xizf -"
+TEST_DEPS="$REL_CHROMA_DIR/chroma-manager/tests/framework/integration/shared_storage_configuration/simulator/test_dependencies"
```
Comment (jgrund) [inline chroma-manager/tests/framework/integration/shared_storage_configuration/simulator/cluster_setup]: Where is $REL_CHROMA_DIR being set?

Comment (brianjmurrell) [inline chroma-manager/tests/framework/integration/shared_storage_configuration/simulator/cluster_setup]: Assuming you found [it](https://github.com/intel-hpdd/intel-manager-for-lustre/pull/18/files/03ce98415b4757c907c315c7cedae6e2da68ceb3#diff-8045fd024ec11b9c80a47e4de53b1945R29) but just want to make sure the question was answered.

File: chroma-manager/tests/framework/integration/installation_and_upgrade/jenkins_steps/main (hunk 1 above)
Comment (brianjmurrell) [inline chroma-manager/tests/framework/integration/installation_and_upgrade/jenkins_steps/main]: So, for the rest of this test, it happens [at](https://github.com/intel-hpdd/intel-manager-for-lustre/commit/de4e8dbd3974d8ad4ef357a8afc1f9fa295ddccc#diff-86e1d83b074882382896e9e2bff4065eR22) but that doesn't help here.

Typically it's set by the test caller, like we do in our Jenkins steps, but it should have a reasonable default set if the caller doesn't set it, to reduce the amount the caller is supposed to know about how the tests run.  I'll push another patch closing up this and any other loophole like this.

**Pull request merged**